CRM/Upgrade/Incremental/sql/4.7.15.mysql.tpl

```diff
@@ -1,4 +1,7 @@
 {* file to handle db changes in 4.7.15 during upgrade *}
 
 -- CRM-19685 (fix for inconsistencies)
-UPDATE civicrm_contact SET preferred_mail_format = 'Both' WHERE preferred_mail_format IS NULL;
\ No newline at end of file
+UPDATE civicrm_contact SET preferred_mail_format = 'Both' WHERE preferred_mail_format IS NULL;
+
+-- CRM-19626
+ALTER TABLE civicrm_price_set ADD min_amount INT(10) UNSIGNED DEFAULT '0' COMMENT 'Minimum Amount required for this set.';
```

Comment:
@jitendrapurohit @yashodha Putting schema changes directly into the sql file means that an upgrade will fail if run twice (e.g. in the case of upgrading an RC -> stable). I've added a utility for creating columns (and we could add more such utilities for other small tasks like this) and moved this code here: #9548